docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+        manager if you believe that your employer might have a corporate
+        account.</para>
+      </listitem>
+
+      <listitem>
+        <para>A code editor of your choice. There are a few editors integrated
+        with Azure such as code, vi the Visual Editor, Nano, or you can choose
+        to use any another.</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>The easiest option which also insures you have all the tools
+    required is to use Azure is the command portal. Assuming you have an Azure
+    account with all the appropriate credentials you can just go to the Azure
+    command portal</para>
+
+    <programlisting>https://portal.azure.com/</programlisting>
+
+    <para>If this is the first time you have accessed the cloud shell, Azure
+    will prompt you that storage is required for the cloud shell to persist
+    account settings and files. Click through the prompts to create the
+    storage. You should be presented with a shell. At this point, the cloud
+    shell should already be logged into to your Azure account. The major
+    advantage to using the cloud shell is that It will also have other
+    prerequisites installed for you, namely, git, kubectl, helm, and even an
+    editor. Otherwise in addition to installing the Azure CLI you would also
+    need to install the other prerequisites.</para>
+
+    <sect2 id="Workin_withTerraform">
+      <title>Working with the Terraform Code</title>
+
+      <para>The HPCC Systems Terraform Repository,
+      <emphasis>terraform-azurerm-hpcc</emphasis> is a code repository where
+      the HPCC Systems Terraform modules are stored. The Terraform repository
+      contains three modules required to instantiate an HPCC Systems cluster.
+      These are the network, storage, and AKS (Azure Kubernetes Service)
+      modules. The network module must be deployed first, followed by the
+      storage module, which is an optional module for deploying a storage
+      account. Finally, the AKS module, which is at the root of the
+      repository, can then be deployed. These modules will automatically call
+      other dependent modules upon initialization. </para>
+
+      <sect3>
+        <title>Cloning the Terraform Repository</title>
+
+        <para>You will want to clone the Terraform Repository,
+        terraform-azurerm-hpcc hosted on the HPCC Systems GitHub
+        account.</para>
+
+        <para><ulink
+        url="https://github.com/hpcc-systems/terraform-azurerm-hpcc.git ">https://github.com/hpcc-systems/terraform-azurerm-hpcc.git</ulink></para>
+
+        <para>To clone the repository:</para>
+
+        <para><orderedlist>
+            <listitem>
+              <para>Open your command line or terminal</para>
+            </listitem>
+
+            <listitem>
+              <para>Determine where to store this project. Choose a location
+              that is easy to remember, such as the Desktop, for example:
```

Comment:
If the instructions are using the Azure cloud shell, then Desktop is not really a valid location. 
I suggest removing the suggestion.